operator/internal/manifests/openshift/options.go

```diff
@@ -36,13 +36,14 @@ type AuthorizationSpec struct {
 // extra lokistack gateway k8s objects (e.g. ServiceAccount, Route, RBAC)
 // on openshift.
 type BuildOptions struct {
-	LokiStackName        string
-	LokiStackNamespace   string
-	GatewayName          string
-	GatewaySvcName       string
-	GatewaySvcTargetPort string
-	RulerName            string
-	Labels               map[string]string
+	LokiStackName          string
+	LokiStackNamespace     string
+	GatewayName            string
+	GatewaySvcName         string
+	GatewaySvcTargetPort   string
+	RulerName              string
+	Labels                 map[string]string
+	OCPAlertManagerEnabled bool
```

Comment:
You can skip the `OCP` prefix here, as the field is accessible via `openshift.BuiltOptions.AlertManagerEnabled`